client/post-editor/editor-sidebar/util.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * External dependencies
+ */
+import { PropTypes } from 'react';
+
+export const NESTED_SIDEBAR_NONE = 'NESTED_SIDEBAR_NONE';
```

Comment:
The whole system around "nested sidebar" seems confusing to me. Could you explain how it works at a higher level and why you chose to implement it this way?